lib/x11utils.pm

```diff
@@ -211,7 +211,7 @@ sub handle_login {
     elsif (match_has_tag('displaymanager-user-prompt') || get_var('DM_NEEDS_USERNAME')) {
         type_string "$myuser\n";
     }
-    elsif (check_var('DESKTOP', 'gnome')) {
+    elsif (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'kde')) {
         if ($user_selected || (is_sle('<15') || is_leap('<15.0'))) {
```

Comment:
Have you verify if version >= 15 then do not need send 'ret' key?

Reply:
The condition has checked '<15', so it won't run those code when version '>=15'. I checked alll leap kde cases, all of them version=15.3. And I checked with desktop group, only 11sp4 use KDE. Other's won't use it. So this change cannot impact other cases.